client/ide/lib/views/contentsearch/idecontentsearchview.coffee

```diff
@@ -0,0 +1,298 @@
+kd                          = require 'kd'
+KDButtonView                = kd.ButtonView
+KDCustomHTMLView            = kd.CustomHTMLView
+KDHitEnterInputView         = kd.HitEnterInputView
+KDMultipleChoice            = kd.MultipleChoice
+JView                       = require 'app/jview'
+$                           = require 'jquery'
+keycode                     = require 'keycode'
+Encoder                     = require 'htmlencode'
+editorSettings              = require '../../workspace/panes/settings/editorsettings'
+IDEContentSearchResultView  = require './idecontentsearchresultview'
+showError                   = require 'app/util/showError'
+showNotification            = require 'app/util/showNotification'
+
+REGEXES =
+  escapeShell      : /([\\"'`$\s\(\)<>])/g
+  escapeRegExp     : /([.*+?\^${}()|\[\]\/\\])/g
+  mainLineRegex    : /^:?([\s\S]+):(\d+):([\s\S]*)$/
+  grepEscapeRegExp : /[[\]{}()*+?.,\\^$|#\s"']/g
+  contextLineRegex : /^([\s\S]+)\-(\d+)\-([\s\S]*)$/
```

Comment:
pls be consistent either call it `regex` or `regexp`, `regex` is more widely used.